Doc/library/io.rst

```diff
@@ -753,9 +754,9 @@ than raw I/O does.
 
 .. class:: BufferedRWPair(reader, writer, buffer_size=DEFAULT_BUFFER_SIZE)
 
-   A buffered I/O object combining two unidirectional :class:`RawIOBase`
-   objects -- one readable, the other writeable -- into a single bidirectional
-   endpoint.  It inherits :class:`BufferedIOBase`.
+   A buffered binary stream over two unidirectional :class:`RawIOBase` raw binary
```

Comment:
```suggestion
   A buffered binary stream with two unidirectional :class:`RawIOBase` raw binary
```

Reply:
Let's use your "providing higher-level access to" phrase here too.